Cargo.toml

```diff
@@ -131,4 +131,5 @@ members = [
 	"miner",
 	"transaction-pool",
 	"whisper",
+	"util/rlp_compress"
```

Comment:
Is this necessary?

Reply:
no, that's leftover from before this crate was used by ethcore ;)